### DISCUSSION TITLE
perda acesso a conta

### DISCUSSION THREAD

**Discussion by NardoniAlice** [community/community#49823] [New to GitHub]
tenho uma conta que foi feita com o email da faculdade. como o acesso ao email não existe mais, não consegui recuperar o acesso. porém tenho repositorios ali que eu gostaria de mesclar a minha nova conta.. 
1- tem como recuperar a conta mesmo não tendo mais acesso ao email institucional usado pra criar a conta?

**Comment by bbkx226** [comment]
If you no longer have access to the email address associated with your old account, it may be difficult to recover that account. If you just want to transfer the repositories from your old account to your new one, you can use GitHub's repository transfer feature. You will need to have admin access to both the old and new repositories. You can find more information on how to transfer a repository on the GitHub Help page.

Translate to Portuguese:
" Se você não tem mais acesso ao endereço de e-mail associado à sua conta antiga, pode ser difícil recuperar essa conta. Se você apenas deseja transferir os repositórios da sua conta antiga para a nova, pode usar o recurso de transferência de repositório do GitHub. Você precisará ter acesso de administrador aos dois repositórios antigo e novo. Você pode encontrar mais informações sobre como transferir um repositório na página de ajuda do GitHub. "

**Comment by NardoniAlice** [comment]
"it may be difficult to recover that account.", but is possible?

**Reply by bbkx226** [reply]
The only way I know is to reach out GitHub Support, since they can manage to solve any unforseen circumstances. Also, if you find this helpful, don't hesitate to mark as answer 😎

**Reply by NardoniAlice** [reply]
e qual o e-mail de contato?

**Reply by bbkx226** [reply]
I think you can refer to this link https://support.github.com/

**Comment by NardoniAlice** [comment]
sim, por esse link eu cheguei na comunidade aqui, mas não achei um e-mail de suporte,tem pra quem paga, pra bugs...

**Reply by bbkx226** [reply]
Kindly refer to this link https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket
